### PR TITLE
Cleanup Comments and Add Change Events

### PR DESCRIPTION
**Pull request by kaliweh-circle**
**Changes:**
- Cleanup the contract comments
- Add missing `ResourceId` interface functions to the `Pauser` and `Blocker`
- Add `AdminChanged`, `OwnerChanged`, `MasterMinterChanged`, `PauserChanged` and `BlocklisterChanged` events
- Cleanup the `OnchainMultiSig` contract
- Remove the `Vault.PublicSigner`
- Remove the `Vault.Allowance`
- Remove the `Minter.PublicSigner`
- Update the Contract init function to create and assign the necessary roles
- Delete broken scripts/transactions that were changed to `OnchainMultiSig` functions
- Stop running all the broken unit tests (Will follow with the test fixes)

### CODE REVIEW

**Review by rheaplex — APPROVED** [review]
This is great! 👍

**Review by walkerq — COMMENTED** [review]
two super minor comments

**Review by walkerq — APPROVED** [review]

File: contracts/FiatToken.cdc [hunk 1]
```diff
@@ -1600,100 +1146,152 @@ pub contract FiatToken: FungibleToken {
...
+                signatureAlgorithm: SignatureAlgorithm(rawValue: blocklisterPubKeysAlgos[i]) ?? panic ("Invalid signature algo")
+            )
+            blocklisterAccount.keys.add(
+                publicKey: key,
+                hashAlgorithm: HashAlgorithm.SHA3_256,
+                weight: blocklisterPubKeysWeights[i]
+            )
+            i = i + 1
+        }
+        blocklisterAccount.save(<- create Blocklister(pk: masterMinterPubKeys, pka: pubKeyAttrs), to: self.BlocklisterStoragePath)
```
Comment (rheaplex) [inline contracts/FiatToken.cdc]: `pk: masterMinterPubKeys`

Should be `blocklisterPubKeys` .

**Review by rheaplex — APPROVED** [review]
This is great! Just one parameter needs correcting.

File: contracts/FiatToken.cdc (hunk 1 above)
Comment (kaliweh-circle) [reply contracts/FiatToken.cdc]: good catch, fixed!

Comment (rheaplex) [reply contracts/FiatToken.cdc]: Awesome!

**Pull request merged**